libraries/picosystem.cpp

```diff
@@ -88,13 +88,19 @@ int main() {
       sleep(20);
     }
   #else
+    // Keep the screen off...
+    backlight(0);
+    // Screen buffer is initialized clear; just flip it.
+    _wait_vsync();
+    _flip();
+    // Wait for the DMA transfer to finish
+    while (_is_flipping());
+    // wait for the screen to update
+    sleep(20);
```

Comment:
Not sure why, but `_wait_vsync()` here gets The Flash™, while just forcing a delay doesn't.  \*shrug\*; there was a 300ms sleep before so whatever.